### PR TITLE
Adds fullscreen support for iOS homescreen

### PR DESCRIPTION
**Pull request by valnub**
When the user pins the app to his homescreen it will run in fullscreen mode (No adressbar, no toolbar) instead of being opened in Safari.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,8 @@
     <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
     <link rel="stylesheet" href="./main.css">
     <meta name="theme-color" content="#ffffff">
+    <meta name="apple-mobile-web-app-capable" content="yes">
```
Comment (Haroenv) [inline index.html]: Already done in #45 😄 

If you pinned it before it’s merged, you need to unpin it and pin it again.

**Comment by samccone** [comment]
awesome thank you!

File: index.html (hunk 1 above)
Comment (valnub) [inline index.html]: Ha, you were faster than me :-)
I also noticed that once the app is in fullscreen mode and code changes are deployed the app cache doesn't like to flush. Any solution for this?
Only workaround seems to be to [add changing numbers](https://stackoverflow.com/questions/8063126/how-can-i-clear-the-cache-of-an-ios-web-app-on-the-homescreen) as GET parameters to referenced sources but that's kinda odd.
Also, it seems like iOS loves to even cache the initial index.html file, so in this case even adding changing numbers to the url won't help `¯\_(ツ)_/¯`

Comment (samccone) [inline index.html]: There is an edge cache from cloudflare .. so from deploy to live takes around 5 min

Comment (valnub) [inline index.html]: @samccone Ok, but I think that's not the problem. It's rather an iOS issue.

**Pull request merged**